src/app/components/templates/Sidebar/SidebarMobile.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+
+import { PropsWithLng } from '@/app/i18next';
+import { useTranslation } from '@/app/i18next/client';
+import DefaultProfile from '@/assets/default-profile.jpeg';
+
+import { User } from '../Navbar/Navbar';
+import Sidebar from '.';
+
+interface SidebarProps {
+  onClose: () => void;
+  user: User | null;
+}
+
+const SidebarMobile = ({ onClose, lng, user }: SidebarProps & PropsWithLng) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const { t } = useTranslation(lng);
+
+  useEffect(() => {
+    setIsOpen(true);
+  }, []);
+
+  const handleClose = () => {
+    setIsOpen(false);
+    setTimeout(onClose, 300);
+  };
```

Comment:
사실 크게 문제가 없긴한데, unmount 되는 경우를 고려한다면 아래 처럼 짜볼 수 있을 것 같아요
```suggestion
  const handleClose = () => {
    setIsOpen(false);
  };
  
  useEffect(() => {
    if (isOpen) return;
    const timer = setTimeout(onClose, 300);
    return () => {
      clearTimeout(timer);
    };
  }, [isOpen]);
```